library/core/tests/iter/adapters/map.rs

```diff
@@ -0,0 +1,49 @@
+use core::iter::*;
+
+#[test]
+fn test_find_map() {
```

Comment:
This is not related to `map` or `Map` -- I think it belongs in the main `traits/iterator.rs`.